admin/grants.sql

```diff
@@ -322,6 +318,7 @@ TO ROLE GENIE_ADMIN;
 -- SYNAPSE_DATA_WAREHOUSE domain roles
 GRANT ROLE SYNAPSE_DATA_WAREHOUSE_ADMIN TO ROLE SYSADMIN;
 GRANT ROLE SYNAPSE_DATA_WAREHOUSE_ANALYST TO ROLE SYNAPSE_DATA_WAREHOUSE_ADMIN;
+GRANT ROLE SYNAPSE_DATA_WAREHOUSE_ANALYST TO ROLE AD;
```

Comment:
This is an attempt at cleaning up the `admin/grants.sql` as we go to better match the final RBAC design (see below - [source](https://sagebionetworks.jira.com/wiki/spaces/DPE/pages/4076863503/Role+and+Privilege+Management)). 

During the initial creation of this file (a commit from 1 year ago), DATA_ANALYTICS was granted to AD, which should have the same effect as what this grant is doing, except DATA_ANALYTICS isn't one of the "core" account roles. I've commented out that grant, in case we decide we want to add it back, and I've added this new grant which is consistent with the design described below:

<img width="735" height="553" alt="image" src="https://github.com/user-attachments/assets/904ff34b-f09d-425f-860e-11c59a065228" />